lua/luasnip/nodes/util/snippet_string.lua

```diff
@@ -130,6 +154,7 @@ function SnippetString:put(pos)
 	end
 end
 
+-- FIXME(@bew): what is the return type here?
```

Comment:
Pretty sure this should just be another `SnippetString`

Reply:
Done in 6204bbd3e99526f766bd029773c5f19145b25ef8